ci-operator/step-registry/ovn/sdn-migration/feature-migration/validate-egress-firewall/ovn-sdn-migration-feature-migration-validate-egress-firewall-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+validate_egressfirewall_cr () {
+  oc get egressfirewall -n test-migration default -o=jsonpath='{.spec}'>current_config
+  echo "$EXPECT_EGRESS_FIREWALL_SPEC" > expected_config
```

Comment:
you might want to dump the output of expected_config to stdout for easier debugging later.

Reply:
fixed.

Reply:
fixed.